kyuubi-common/src/test/scala/org/apache/kyuubi/TestUtils.scala

```diff
@@ -92,11 +92,13 @@ object TestUtils {
     } else {
       val linesInFile = Files.readAllLines(markdown, StandardCharsets.UTF_8)
       val formatted = formatMarkdown(newOutput)
-      val hint = s"$markdown out of date, please update doc with " +
-        s"KYUUBI_UPDATE=1 build/mvn clean install -Pflink-provided,spark-provided,hive-provided " +
-        s"-DwildcardSuites=$agent"
-      assert(linesInFile.size() === formatted.size, hint)
-      linesInFile.asScala.zip(formatted).foreach { case (out, in) => assert(out === in, hint) }
+      linesInFile.asScala.zipWithIndex.zip(formatted).foreach { case ((str1, index), str2) =>
+        withClue(s"$markdown out of date, as line ${index + 1} is not expected." +
+          s" Please update doc with KYUUBI_UPDATE=1 build/mvn clean install" +
+          s" -Pflink-provided,spark-provided,hive-provided -DwildcardSuites=$agent") {
```

Comment:
The following command should be sufficient and faster
```
KYUUBI_UPDATE=1 build/mvn clean test -Pflink-provided,spark-provided,hive-provided  -pl kyuubi-server -am -DwildcardSuites=org.apache.kyuubi.config.AllKyuubiConfiguration
```

Reply:
Addressed.

Reply:
Please search "AllKyuubiConfiguration" globally and make sure all places are updated.

Reply:
OK.